code/modules/mob/living/simple_animal/abnormality/waw/generalb.dm

```diff
@@ -0,0 +1,168 @@
+/mob/living/simple_animal/hostile/abnormality/general_b
+	name = "General Bee"
+	desc = "A bee humanoid creature."
+	icon = 'ModularTegustation/Teguicons/48x96.dmi'
+	icon_state = "generalbee"
+	icon_living = "generalbee"
+	faction = list("hostile")
+	speak_emote = list("buzzes")
+	pixel_x = -8
+	base_pixel_x = -8
+	threat_level = WAW_LEVEL
+	start_qliphoth = 1
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = list(0, 0, 55, 55, 60),
+						ABNORMALITY_WORK_INSIGHT = list(0, 0, 45, 45, 50),
+						ABNORMALITY_WORK_ATTACHMENT = 0,						//DO NOT FUCK THE BEEGIRL
```

Comment:
This is not what attachment is :pensive:

Reply:
She's a hardened war criminal who bombs children.
No time to sit down and chat.

Reply:
You will not announce when you are performing Attachment Work with the Anime Bee Girl for she will have murdered you. 👍